source/client/Controller.py

```diff
@@ -118,6 +121,11 @@ def getHand(self):
         """sends _state to UI"""
         return self._state.hand_cards.copy()
 
+    def getPreparedCards(self):
+        """lets the UI fetch prepared cards"""
+        #TODO: Sheri - should this information be returned in getHand or is a separate method best?
```

Comment:
re: my TODO:  I'll look at when getHand is called to determine whether GetPreparedCards should be included in getHand.